texts/MANIFEST

```diff
@@ -0,0 +1,5 @@
+1   disco       dolphingarlic
+1   happiness   dolphingarlic
+1   icarus      dolphingarlic
+1   kubla_khan  dolphingarlic
+1   reggae      dolphingarlic
```

Comment:
If you do it like this, people will think you created all these texts by yourself.  You need to say where you get the texts, and ideally provide some information about each one (year, style, author, license, etc.).

Reply:
I added author, style, and where I got them from now